app/view_objects/result_filters/location_view.rb

```diff
@@ -0,0 +1,23 @@
+module ResultFilters
+  class LocationView
```

Comment:
They way this is implemented, it's closer to a form object than a view object. A form object is meant to abstract logic that's relevant to inputs sent into the app from a form. For example, we could've used that for the create wizard to have validations that are specific to the way the form is used (we could've used that for the modern language subjects issue we looked at recently).

A view object is for logic that is specific to the view, instead of using helpers, for example. We mentioned presenters also, those usually wrap models in a way to make them presentable in views (aka decorators, we already use these via the gem `draper` in publish).

I think the confusion comes partially from the way `ResultsFilter::QualificationView` is implemented, tbh (sorry @gpeng :) ). It blurs the lines a little bit by providing helper-style functionality that inspects the params.

Reply:
I have removed this file and put the params helper methods back into the LocationController